src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +45,8 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream().filter(number -> number > 3)
+                .map(number -> number * 2)
+                .reduce(0, Integer::sum);
```

Comment:
사소하지만 이렇게도 바꿔볼 수 있겠네요 😄 
```suggestion
               .mapToInt(number -> number * 2)
               .sum();
```